### PR TITLE
Vajdalilla list authentications

### PR DESCRIPTION
**Pull request by adamgyulavari**

### CODE REVIEW

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/AuthController.java [hunk 1]
```diff
@@ -35,4 +35,14 @@ public AuthResponse postAuth(@RequestBody AuthModel authModel, @RequestHeader("X
 
         return new AuthResponse(userModel.getId(), userModel.getClientToken(), authModel.getEmail(), authModel.getAccessToken());
     }
+
+    @GetMapping("/auth")
+    public ResponseEntity<?> listAuthentications(@RequestHeader("X-Client-Token") String clientToken, HttpServletRequest request) throws IOException {
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/AuthController.java]: @vajdalil this method should be named like `methodTypeEndpoint`, so `getAuth`

File: src/main/resources/application.properties.example [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-
-#Google Calendar API Configuration
-google.client.client-id=${CLIENT_ID}
-google.client.client-secret=${CLIENT_SECRET}
-google.client.access-token-uri=https://www.googleapis.com/oauth2/v3/token
-google.client.user-authorization-uri=https://accounts.google.com/o/oauth2/auth?access_type=offline&prompt=consent
-google.client.client-authentication-scheme=query
-google.client.scope=profile,email,https://www.googleapis.com/auth/calendar
-google.resource.user-info-uri=https://www.googleapis.com/oauth2/v2/userinfo
-google.resource.prefer-token-info=true
```
Comment (adamgyulavari) [inline src/main/resources/application.properties.example]: @vajdalil pls make it back!1!!

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/AuthController.java [hunk 3]
```diff
@@ -1,24 +1,34 @@
...
     @Autowired
     AuthAndUserService authAndUserService;
 
+    @GetMapping("/auth")
+    public ResponseEntity getAuth(@RequestHeader("X-Client-Token") String clientToken, HttpServletRequest request) throws IOException {
+        if (!request.getHeader("X-Client-Token").equals("")) {
+            UserModel userModel = authAndUserService.findUserByClientToken(clientToken);
+            return ResponseEntity.ok().body(new UserResponse(userModel.getId(), userModel.getUserEmail(), userModel.getAuthModelList()));
+        }
+        return ResponseEntity.badRequest().body("Client token is missing or invalid");
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/AuthController.java]: @vajdalil lets have `unauthorized` http status here

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/AuthController.java [hunk 4]
```diff
@@ -28,11 +38,13 @@ public AuthResponse postAuth(@RequestBody AuthModel authModel, @RequestHeader("X
...
-            userModel.setId(userModel.getId());
         }
         authModel.setUser(userModel);
         authAndUserService.saveAuthModel(authModel);
 
         return new AuthResponse(userModel.getId(), userModel.getClientToken(), authModel.getEmail(), authModel.getAccessToken());
     }
+
+
+
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/AuthController.java]: @vajdalil I suppose we dont need these

**Review by gumimaci88 — APPROVED** [review]

**Pull request merged**